README.md

```diff
@@ -22,17 +22,25 @@ Now scaling is easy!
 
 BREAKING CHANGES since [version 0.3.0](https://github.com/wemake-services/caddy-gen/releases/tag/0.3.0)!
 
-Options to configure:
+Main options to configure:
 
 - `virtual.host` domain name, don't pass `http://` or `https://`, you can separate them with space,
 - `virtual.alias` domain alias, e.q. `www` prefix,
 - `virtual.port` port exposed by container, e.g. `3000` for React apps in development,
 - `virtual.tls-email` the email address to use for the ACME account managing the site's certificates,
+- `virtual.host.directives` set custom [Caddyfile directives](https://caddyserver.com/docs/caddyfile/directives) for the host. Separate multiple directives with newline characters ("\n").
+- `virtual.host.import` import Caddyfile directives for the host from a file on the container's filesystem.
+
+Basic authentication options:
 - `virtual.auth.path` with
 - `virtual.auth.username` and
-- `virtual.auth.password` together provide HTTP basic authentication.
+- `virtual.auth.password` together provide HTTP basic authentication. (Password should be a string `base64` encoded from `bcrypt` hash. You can use https://bcrypt-generator.com/ with default config and https://www.base64encode.org/.)
 
-Password should be a string `base64` encoded from `bcrypt` hash. You can use https://bcrypt-generator.com/ with default config and https://www.base64encode.org/.
+Reverse proxy options:
+- `virtual.proxy.matcher` have the reverse proxy only match certain paths.
```

Comment:
Can we use it to serve files like you have mentioned in #48 ?
If so, let's add it to the docs! 👍

Reply:
Good idea, added here https://github.com/wemake-services/caddy-gen/pull/52/commits/354ebfd8cc360f9f7068830011653380dcc15119?short_path=b335630#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5